styles/linter-plus.less

```diff
@@ -15,6 +15,9 @@ linter-message {
   overflow-y: scroll;
   max-height: 150px;
   padding: 4px 8px;
+  &[hidden]{
```

Comment:
native styling of `[hidden]{display:none}` is overridden when we use a more direct selector and set `display:block`.